diff --git a/node_modules/react-native-vosk/src/NativeVosk.ts b/node_modules/react-native-vosk/src/NativeVosk.ts
index fe1dce5..68740ab 100644
--- a/node_modules/react-native-vosk/src/NativeVosk.ts
+++ b/node_modules/react-native-vosk/src/NativeVosk.ts
@@ -22,12 +22,6 @@ export interface Spec extends TurboModule {
 
   addListener: (eventType: string) => void;
   removeListeners: (count: number) => void;
-
-  readonly onResult: CodegenTypes.EventEmitter<string>;
-  readonly onPartialResult: CodegenTypes.EventEmitter<string>;
-  readonly onFinalResult: CodegenTypes.EventEmitter<string>;
-  readonly onError: CodegenTypes.EventEmitter<string>;
-  readonly onTimeout: CodegenTypes.EventEmitter<void>;
 }
 
 export default TurboModuleRegistry.getEnforcing<Spec>('Vosk');
